BE/LegalEntities/FormalBusinessOrganizations.rdf

```diff
@@ -209,12 +213,63 @@
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>organizational sub-unit</rdfs:label>
-		<skos:definition>any department, service, or other entity within a larger organization that only has full recognition within the context of that organization, but requires identification for some purpose</skos:definition>
+		<skos:definition>any department, service, and other entity within a larger organization that only has full recognition within the context of that organization, but requires identification for some purpose</skos:definition>
 		<fibo-fnd-utl-av:adaptedFrom rdf:datatype="&xsd;anyURI">https://www.iso.org/obp/ui/#iso:std:iso-iec:6523:-1:ed-1:v1:en</fibo-fnd-utl-av:adaptedFrom>
 		<fibo-fnd-utl-av:explanatoryNote>In other words, it is not a legal entity in its own right.</fibo-fnd-utl-av:explanatoryNote>
 		<fibo-fnd-utl-av:synonym>organization part</fibo-fnd-utl-av:synonym>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-be-le-fbo;TaxIdentificationScheme">
+		<rdfs:subClassOf rdf:resource="&lcc-lr;IdentificationScheme"/>
+		<rdfs:label>tax identification scheme</rdfs:label>
+		<skos:definition>identification scheme dedicated to the unique identification of taxpayers in some jurisdiction</skos:definition>
```

Comment:
"dedicated" is wrong, since in the US SSN and EIN are not solely for tax purposes. 
Also "taxpayer" is wrong since many people with a SSN do not pay tax, either because they are ineligible (e.g. due to age or where theyare located -only US taxes citizens not resident in the US) or do not earn enough.

Reply:
An SSN is NOT specifically a tax id - it is a national identification number that happens to be used as a tax id, but that was not its original purpose.  It is not issued by the IRS - a TIN, EIN, and adoption identifiers are issued by the IRS, not an SSN.